ui/src/components/shared/Router.tsx

```diff
@@ -50,15 +62,29 @@ export function Router() {
 
   return (
     <Routes>
-      <Route element={<Navigate to={ROUTES.schemas.path} />} path={ROOT_PATH} />
+      {identifier ? (
+        <>
+          <Route element={<Navigate to={ROUTES.schemas.path} />} path={ROOT_PATH} />
+
+          <Route element={<FullWidthLayout />}>{getLayoutRoutes("fullWidth")}</Route>
 
-      <Route element={<FullWidthLayout />}>{getLayoutRoutes("fullWidth")}</Route>
+          <Route element={<FullWidthLayout background="bg-light" />}>
+            {getLayoutRoutes("fullWidthGrey")}
+          </Route>
 
-      <Route element={<FullWidthLayout background="bg-light" />}>
-        {getLayoutRoutes("fullWidthGrey")}
-      </Route>
+          <Route element={<SiderLayout />}>{getLayoutRoutes("sider")}</Route>
+        </>
+      ) : (
+        <>
+          <Route element={<SiderLayout />}>
+            {issuerRoutes.map(({ Component, path }) => (
+              <Route element={<Component />} key={path} path={path} />
+            ))}
+          </Route>
```

Comment:
Instead of creating the `issuerRoutes` list above and then mapping over it here, since this is a very specific and special case I think I would declare the routes here directly to make the code more cohesive and easier to read:

```typescript
<Route element={<SiderLayout />}>
  <Route
    element={<COMPONENTS.issuers />}
    key={ROUTES.issuers.path}
    path={ROUTES.issuers.path}
  />
  <Route
    element={<COMPONENTS.createIssuer />}
    key={ROUTES.createIssuer.path}
    path={ROUTES.createIssuer.path}
  />
</Route>
```

By the way, how is it that we allow navigating to `ROUTES.issuers` when there's no `identifier`? Shouldn't we limit the valid routes to `ROUTES.createIssuer` until `identifier` is available?

Reply:
okay i will change this part.

for the `ROUTES.createIssuer` route we have a representation for the case where no issuers are created